searchcore/src/vespa/searchcore/proton/attribute/attribute_writer.cpp

```diff
@@ -177,6 +178,39 @@ applyCompactLidSpace(uint32_t wantedLidLimit, SerialNum serialNum,
     }
 }
 
+using AttrUpdates = std::vector<std::pair<AttributeVector *, const FieldUpdate *>>;
+
+struct BatchUpdateTask : public vespalib::Executor::Task {
+
+    BatchUpdateTask(SerialNum serialNum, DocumentIdT lid, bool immediateCommit,
+                    AttributeWriter::OnWriteDoneType onWriteDone)
+            : AttrUpdates(),
```

Comment:
Missing explicit initialization of _updates().
Missing initialization for parent class (vespalib::Executor::Task())
Leftover initialization of previous parent class (AttrUpdates)